Userland/Libraries/LibWeb/CSS/MediaQuery.cpp

```diff
@@ -392,60 +392,6 @@ String serialize_a_media_query_list(Vector<NonnullRefPtr<MediaQuery>> const& med
     return MUST(String::join(", "sv, media_queries));
 }
 
-bool is_media_feature_name(StringView name)
```

Comment:
:laughing: Nice catch! Can't believe this hung around for 2 years.

Reply:
I must admit - I realised it was no longer used, after resolving the FIXME. :cry:

Reply:
Sorry! :sweat_smile: